sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -799,10 +808,10 @@ void CAFMaker::produce(art::Event& evt) noexcept {
           }
           const unsigned int idx = fWeightPSetIndex[it.first];
           if(idx >= srneutrinos.back().wgt.size()) srneutrinos.back().wgt.resize(idx+1);
-	  srneutrinos.back().wgt[idx].univ = it.second;
+          srneutrinos.back().wgt[idx].univ = it.second;
```

Comment:
Filling the SR objects really shouldn't go in CAFMaker_module itself -- it should be split out into one of the functions inside FillTrue.

Reply:
I pushed a couple of commits moving things to FillTrue